dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -297,6 +305,14 @@ public String getExtensionName(Class<?> extensionClass) {
                         loadedExtensions.clear();
                     }
                 } else {
+                    if (notMatchFilters.contains(name)) {
```

Comment:
if a filter config dubbo.consumer.filter, the filter always be notMatch, whatever the group is match or not match.